plugin.cmake

```diff
@@ -68,6 +69,15 @@ set_property(TEST server_large_image.cached_tiles.PythonCache APPEND PROPERTY
   "LARGE_IMAGE_CACHE_BACKEND=python"
   "LARGE_IMAGE_DATA=${PROJECT_BINARY_DIR}/data/plugins/large_image")
 
+add_python_test(sources PLUGIN large_image BIND_SERVER EXTERNAL_DATA
+  # I don't know why adding this data file thows a cmake error.
+  # "plugins/large_image/sample_image.ptif"
```

Comment:
I don't understand why uncommenting this external data file results in a cmake error.  The error is
```
CMake Error: Attempt to add a custom rule to output "/home/ubuntu/misc/large_image/_build/data/plugins/large_image/sample_image.ptif.sha512-stamp.rule" which already has a custom rule.
```
I can use other external data files in multiple tests, but this throws an error.

@zachmullen Do you know what this is happening, or who I should ask?